### ISSUE TITLE
docker部署出现的问题，不知如何解决

### ISSUE THREAD

**Issue by jianglianwen** [mlogclub/bbs-go#186]
![验证码](https://user-images.githubusercontent.com/73823366/208288908-5e86a955-a476-4981-84e3-ea2eb0f45512.png)

![bbs01](https://user-images.githubusercontent.com/73823366/208288915-a5c636a0-1f8c-483b-ba39-8d1bf60cc538.png)
![bbs-go docker yaml](https://user-images.githubusercontent.com/73823366/208288917-e54c27bc-533d-4b0c-9ad2-5414dcfa88f6.png)

**Comment by dandyhuang** [comment]
> ![验证码](https://user-images.githubusercontent.com/73823366/208288908-5e86a955-a476-4981-84e3-ea2eb0f45512.png)
> 
> ![bbs01](https://user-images.githubusercontent.com/73823366/208288915-a5c636a0-1f8c-483b-ba39-8d1bf60cc538.png) ![bbs-go docker yaml](https://user-images.githubusercontent.com/73823366/208288917-e54c27bc-533d-4b0c-9ad2-5414dcfa88f6.png)

我看你请求的是localhost:3000的。没有请求的配置的公网的ip

**Comment by jianglianwen** [comment]
> > ![验证码](https://user-images.githubusercontent.com/73823366/208288908-5e86a955-a476-4981-84e3-ea2eb0f45512.png)
> > ![bbs01](https://user-images.githubusercontent.com/73823366/208288915-a5c636a0-1f8c-483b-ba39-8d1bf60cc538.png) ![bbs-go docker yaml](https://user-images.githubusercontent.com/73823366/208288917-e54c27bc-533d-4b0c-9ad2-5414dcfa88f6.png)
> 
> 我看你请求的是localhost:3000的。没有请求的配置的公网的ip

对 这也是我所奇怪的地方，我在server中的bbs-go.docker.yaml中已经配置过公网的ip:3000了也正如我截图中所示，但是浏览器中仍旧是localhost，而且在admin中我的验证码是生效的，只有site中的是无效的，我很困惑。

**Comment by dandyhuang** [comment]
看一下你的nuxt.config.js配置

**Comment by jianglianwen** [comment]
nuxt.config.dokcer.js中我只改了这部分内容
![image](https://user-images.githubusercontent.com/73823366/208613071-494cd595-069f-4d96-bb3e-60195b3d505b.png)

**Comment by dandyhuang** [comment]
你看一下你的server服务是不是报错了
<img width="993" alt="image" src="https://user-images.githubusercontent.com/12603054/208613852-6205755c-96a8-48a6-b9fc-ed3b42b965ea.png">
加载验证码的时候

**Comment by jianglianwen** [comment]
我并没有在本地环境搭建，但是验证码这里admin在进行登录操作时是可用的。
![image](https://user-images.githubusercontent.com/73823366/208615904-b46389f8-98ad-4cb9-8ec4-9c34612dd91d.png)
![image](https://user-images.githubusercontent.com/73823366/208615970-7222fd01-8ca6-4cf4-a8bb-1efab76a0cd0.png)

**Comment by dandyhuang** [comment]
BaseUrl修改完后，服务重启了没。可以在sever或者site里头加一下debug看看

**Comment by jianglianwen** [comment]
问题解决了 在server的容器中 有一个bbs-go.yaml 在这里的BaseUrl是localhost我给改掉了

**Comment by jianglianwen** [comment]
在注册用户时，数据库中score这个字段会报错
![image](https://user-images.githubusercontent.com/73823366/208619862-5c2fca2a-77fc-446c-b1d2-a259ffcb60d6.png)

**Comment by dandyhuang** [comment]
> 问题解决了 在server的容器中 有一个bbs-go.yaml 在这里的BaseUrl是localhost我给改掉了
那你之前确认错了

**Comment by jianglianwen** [comment]
> > 问题解决了 在server的容器中 有一个bbs-go.yaml 在这里的BaseUrl是localhost我给改掉了
> > 那你之前确认错了

**Comment by jianglianwen** [comment]
注册用户时，默认不会给score值设为0，这个是我哪个步骤搞错了嘛
![image](https://user-images.githubusercontent.com/73823366/208622662-247d59b4-9a7b-4315-97b1-a4c13321d71a.png)

**Comment by dandyhuang** [comment]
> 注册用户时，默认不会给score值设为0，这个是我哪个步骤搞错了嘛 ![image](https://user-images.githubusercontent.com/73823366/208622662-247d59b4-9a7b-4315-97b1-a4c13321d71a.png)

你是要干嘛还是啥呢

**Comment by jianglianwen** [comment]
> > 注册用户时，默认不会给score值设为0，这个是我哪个步骤搞错了嘛 ![image](https://user-images.githubusercontent.com/73823366/208622662-247d59b4-9a7b-4315-97b1-a4c13321d71a.png)
> 
> 你是要干嘛还是啥呢

我要添加用户，但是添加不了

**Comment by dandyhuang** [comment]
我意思你要搭这个项目干嘛呢
